projects/bitcoin-core/project.yaml

```diff
@@ -0,0 +1,14 @@
+homepage: "https://github.com/bitcoin/bitcoin"
+language: c++
+primary_contact: "marco@chaincode.com"
+auto_ccs:
+  - "jonas@chaincode.com"
+  - "evan@chaincoe.com"
+  - "fanquake@gmail.com"
+  - "practicalswift?"
```

Comment:
@practicalswift Do you want to be added here or not?

Also is there anyone else who would like to be added?

Reply:
To add:

```
   - "laanwj@gmail.com"
   - "pieter@chaincode.com"

Reply:
@MarcoFalke Sure, feel free to include my address `thomas.j.bitcoin@protonmail.com` if you think I can be of any help. (If we want to keep down the number of `auto_ccs` I'm perfectly fine with being excluded too: I'm super happy simply by the fact that our code will get more thoroughly fuzzed :))

Reply:
No obligation to help if you put your mail address there. I believe it is some kind of read-only mailing list subscription. You can also browse the issues in the browser without login, if you prefer. Only security issues may be limited for public consumption temporarily.

Added you for now. Happy to add or remove anyone else.